src/main/java/org/sonarsource/sonarlint/ls/backend/BackendService.java

```diff
@@ -187,6 +187,12 @@ public CompletableFuture<GetBindingSuggestionsResponse> getBindingSuggestion(Get
     return initializedBackend().getBindingService().getBindingSuggestions(params);
   }
 
+  public CompletableFuture<org.sonarsource.sonarlint.core.clientapi.backend.issue.CheckStatusChangePermittedResponse>
+  checkStatusChangePermitted(String connectionId, String issueKey) {
+    return initializedBackend().getIssueService().checkStatusChangePermitted(
+      new org.sonarsource.sonarlint.core.clientapi.backend.issue.CheckStatusChangePermittedParams(connectionId, issueKey));
```

Comment:
Not introduced in this PR, but I wonder if we could have 'CheckIssueStatusChangePermittedParams' and   'CheckHotspotStatusChangePermittedParams'  classes so we avoid using the fully classified paths?
And similar for respective Response classes?